triggers/tests/BUILD.bazel

```diff
@@ -210,9 +171,61 @@ EOF
                 "//triggers/runner:trigger-runner-lib",
             ],
         ),
+    ] + [
+        [
+            da_scala_test_suite(
+                name = "trigger-integration-tests" + test_suffix + version_suffix,
+                srcs = test_suite,
+                data = [
+                    ":acs%s.dar" % version_suffix,
+                ],
+                resources = ["//triggers/runner:src/main/resources/logback.xml"],
+                scala_deps = [
+                    "@maven//:com_typesafe_akka_akka_stream",
+                    "@maven//:org_scalacheck_scalacheck",
+                    "@maven//:org_scalatestplus_scalacheck_1_15",
+                    "@maven//:org_scalaz_scalaz_core",
+                ],
+                tags = ["cpu:4"] + test_tag,
+                deps = [
+                    ":test-utils",
+                    "//bazel_tools/runfiles:scala_runfiles",
+                    "//canton/it-lib",
+                    "//daml-lf/archive:daml_lf_archive_reader",
+                    "//daml-lf/data",
+                    "//daml-lf/engine",
+                    "//daml-lf/interpreter",
+                    "//daml-lf/language",
+                    "//language-support/scala/bindings",
+                    "//language-support/scala/bindings-akka",
+                    "//ledger-api/rs-grpc-bridge",
+                    "//ledger-api/testing-utils",
+                    "//ledger/ledger-api-auth",
+                    "//ledger/ledger-api-common",
+                    "//libs-scala/caching",
+                    "//libs-scala/contextualized-logging",
+                    "//libs-scala/ledger-resources",
+                    "//libs-scala/logging-entries",
+                    "//libs-scala/ports",
+                    "//libs-scala/resources",
+                    "//libs-scala/scalatest-utils",
+                    "//observability/tracing",
+                    "//test-common",
+                    "//triggers/runner:trigger-runner-lib",
+                ],
+            ),
+        ]
+        for (test_suite, test_tag, test_suffix) in [
+            (TRIGGER_INTEGRATION_TESTS, [], ""),
+            (
+                TRIGGER_CANTON_INTEGRATION_TESTS,
+                ["dev-canton-test"],
```

Comment:
```suggestion
                ["dev-canton-test"] if lf_version == "1.dev" else [],
```

Reply:
`dev-canton-test` are needed for those tests that actually use `//canton:canton-patched`. In practice a test need to use `//canton:canton-patched` instead of normal `//canton:canton` only if it requires canton running LF 1.dev (Those tests using stable version of LF should be fine with `//canton:canton`)